src/common_modules/vsts-cli-common/vsts/cli/common/exception_handling.py

```diff
@@ -12,9 +12,9 @@
 logger = get_logger(__name__)
 
 def handle_command_exception(exception):
```

Comment:
This small change will cause only non user errors to have their callstack emitted.
By defining our own exception_handler on VstsCLI, I could go a step further and nuke the ugly boilerplate try/except blocks on each command that call this.

Knack already has logic for emitting CLIError as an error rather than exception: https://github.com/Microsoft/knack/blob/master/knack/cli.py#L159